crypto/bls12381_utils.go

```diff
@@ -127,6 +127,7 @@ func writeScalar(dest []byte, x *scalar) {
 
 // readScalar reads a scalar from a slice of bytes
 func readScalar(x *scalar, src []byte) {
+	C.bn_make((*C.bn_st)(x), (C.int)(prKeyLengthBLSBLS12381))
```

Comment:
The breaking change is coming from this [change](https://github.com/relic-toolkit/relic/commit/16f3c55b055117e1e4cc82160675e2a778a2ed6e#diff-571ef5a4207a1516c00f005d17d24201ea5c28a7f4db104da2605adc1a53086f) in the trimming function. One way to solve it is indeed to force a `bn_new` call (maps to `bn_make`). 

It would be good to have the function assume the parameters are properly initialized and force the caller to initialize variables before calling the function. I have implemented a suggestion [here](https://github.com/onflow/flow-go/tree/tarak/update-relic-version).